ui/src/shared/copy/notifications.ts

```diff
@@ -836,3 +836,8 @@ export const fluxResponseTruncatedError = (): Notification => {
     message: `Large response truncated to first ${APPROX_MAX_RESPONSE_MB} MB`,
   }
 }
+
+export const csvExportFailed: Notification = {
+  ...defaultErrorNotification,
+  message: 'CSV Export failed',
```

Comment:
I was weighing three options:

1. Display any error thrown any where in the stack to the user
2. Wrap a lot of the `downloadTimeseriesCSV` logic in try / catch blocks that re-throw errors with a more user friendly message (i.e. Go style)
3. Not displaying specific errors

I felt (1) wasn't useful and (2) would have added a lot of boilerplate to the `downloadTimeseriesCSV` implementation for not much benefit, since it would be an unrecoverable error. So I opted for (3), but I'm not particularly set on it. What do you think?

Reply:
I see. Number three seems fine then.